design/merge-patch-and-strategic-in-resource-modifier.md

```diff
@@ -88,25 +83,18 @@ Example of StrategicPatches in ResourceModifierRule
 version: v1
 resourceModifierRules:
 - conditions:
-  groupResource: pods
-  resourceNameRegex: "^my-pod$"
-  namespaces:
-  - ns1
+    groupResource: pods
+    resourceNameRegex: "^my-pod$"
+    namespaces:
+    - ns1
   strategicPatches:
-  - patchData: |
```

Comment:
don't we need this for yaml as well?

Reply:
yes

Reply:
I meant the "|"
please add it back

Reply:
I am wondering if we make the field bytearray wouldn't it be possible to support either json or yaml here?
Irrespective of if the RM configmap is built as a json or a yaml
If yes I would prefer that route.

Reply:
RM configmap yaml Marshal will fail if you use []byte